core/proxy.go

```diff
@@ -130,6 +130,10 @@ func NewWebserverProxy(hoverfly *Hoverfly) *goproxy.ProxyHttpServer {
 		w.WriteHeader(resp.StatusCode)
 		w.Write([]byte(body))
 
+		if hoverfly.Cfg.LogHttpRequestResponse {
```

Comment:
this will only log for `webserver` mode. Do you need the log for proxy mode?

Reply:
In our case, we just use it in `webserver` mode during running test cases. The simulation file is created manually.
In proxy mode, the captured results are exported as simulation file, which may be similar to the journals. I don't know if we need logging them in real time.

Reply:
fair, if you only use `webserver` mode